Updated token reference - acr / tfp claims

craigbrown opened the pull request:
The token reference document contains a paragraph explaining that the policy name is returned in the `acr` claim. Further up, the same document recommends using the `tfp` claim for this instead.

I've changed the wording of that paragraph to reflect this.

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

PRmerger commented:
@alexandair : Thanks for your contribution to the Azure documentation! The author, @richrundmsft, has been notified to review your proposed change.

richrundmsft commented:
#sign-off

Commit:
Updated token reference - acr / tfp claims

The token reference document contains a paragraph explaining that the policy name is returned in the `acr` claim. Further up, the same document recommends using the `tfp` claim for this instead.

I've changed the wording of that paragraph to reflect this.
